### PR TITLE
Migrate WCPay shopper tracks to wcpay prefix

### PR DESCRIPTION
**Pull request by malithsen**
Related p2: paJDYF-9HR-p2
Related WooPay issue: 1907-gh-Automattic/woopay

#### Changes proposed in this Pull Request
- Rename some shopper-originated Tracks events and migrate them to `wcpay` prefix.
- Track `wcpay_checkout_page_view` and `wcpay_checkout_order_placed` on all WCPay stores.
- Make sure `checkout_order_placed` is only tracked on non WooPay checkouts.
- Introduce `theme_type` and `woopay_enabled` props to `wcpay_checkout_page_view` event.

#### Testing instructions

**wcpay_woopay_button_click**
* Make sure WooPay Express Button is enabled in the cart, product, and checkout pages.
* As a non-admin user, add a product to the chart, visit the checkout page, and click the WooPay button.
* In ~5 minutes, search MC's Tracks live for the `wcpay_woopay_button_click` event.
* Ensure "Filter for only rejected events" checkbox is selected. Because `wcpay` prefix is still unregistered, these events are currently rejected. 
* Make sure there's an event from your test blog containing the `source: checkout` property.
* Redo the test for button in cart and product pages and ensure the `source` property contains the correct value.

**wcpay_checkout_order_placed**
* Place a standard checkout order (without using WooPay)
* In ~5 minutes, search MC live view for `wcpay_checkout_order_placed` and ensure an event exists.
* Place a WooPay order
* In ~5 minutes, search MC live view for `wcpay_checkout_order_placed` and ensure an event does not appear.

**wcpay_checkout_page_view**
* Make sure WooPay is enabled on the store
* As a non-admin user, add a product to the cart, and go to the classic checkout page.
* Then view the Blocks checkout page.
* In ~5 minutes, search MC live view for `wcpay_checkout_page_view` two events.
* Make sure one event has `woopay_enabled: true` and `theme_type: short_code` properties.
* The other event has `woopay_enabled: true` and `theme_type: blocks` properties.
* Disable WooPay.
* Repeat the steps from 2nd step.
* Ensure that `woopay_enabled: false` property is present in the two events.

-------------------

- [x] Run `npm run changelog` to add a changelog file, choose `patch` to leave it empty if the change is not significant. You can add multiple changelog files in one PR by running this command a few times. 
- [ ] Covered with tests (or have a good reason not to test in description ☝️)
- [ ] Tested on mobile (or does not apply)

**Post merge**

- [ ] Link to testing instructions from [release testing doc](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions) following [these instructions](https://github.com/Automattic/woocommerce-payments/wiki/How-to-write-good-manual-testing-scenarios) : _Add link here / 'QA Testing Not Applicable'_
- [ ] Add or update [critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Critical-flows) and [testing instructions for critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Testing-instructions-for-critical-flows), if applicable.
- [ ] Add what's changed (description, screenshot, demo videos etc.) to the release announcement post, if applicable.

### CODE REVIEW

**Comment by botwoo** [comment]
### Test the build 

#### Option 1. Jetpack Beta

- Install and activate [Jetpack Beta](https://jetpack.com/download-jetpack-beta/).
- Use this build by searching for PR number `6870` or branch name `add/track-wcpay-shopper-events` in your-test.site/wp-admin/admin.php?page=jetpack-beta&plugin=woocommerce-payments          

#### Option 2. Jurassic Ninja - available for logged-in A12s

:rocket: [Launch a JN site with this branch](https://jurassic.ninja/create/?jetpack-beta&shortlived&nojetpack&woocommerce&branches.woocommerce-payments=add/track-wcpay-shopper-events) :rocket:

:information_source: Install this [Tampermonkey script](https://github.com/Automattic/woocommerce-payments/tree/develop/bin/wcpay-live-branches) to get more options.

---

Build info: 

- Latest commit: a6dac21e3015fd9f7ff85ee6c99786a48a7de93b
- Build time: 2023-07-28 21:35:06 UTC

_Note: the build is updated when a new commit is pushed to this PR._

File: includes/class-woopay-tracker.php [hunk 1]
```diff
@@ -163,7 +194,12 @@ public function should_enable_tracking( $is_admin_event = false ) {
 			return false;
 		}
 
-		// Don't track when woopay is disabled.
+		// Track without checking for .
```
Comment (alefesouza) [inline includes/class-woopay-tracker.php]: This comment looks incomplete.

**Review by alefesouza — APPROVED** [review]
LGTM, just a single nit.

File: includes/class-woopay-tracker.php (hunk 1 above)
Comment (malithsen) [reply includes/class-woopay-tracker.php]: Ooh 🤦 Fixed in https://github.com/Automattic/woocommerce-payments/pull/6870/commits/a6dac21e3015fd9f7ff85ee6c99786a48a7de93b

Thanks Alefe

**Pull request merged**